Libraries/WebSocket/WebSocket.windows.js

```diff
@@ -1,27 +1,52 @@
 /**
  * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
  *
```

Comment:
Detailed review needed.

Reply:
RNTester WebSocket example still working (tested both binary and text send/receive)


---
In reply to: [169649951](https://github.com/Microsoft/react-native-windows/pull/1659#discussion_r169649951) [](ancestors = 169649951)